client/app/tasks/hooks/useDeleteTask.ts

```diff
@@ -0,0 +1,40 @@
+import { useRouter } from 'next/navigation'
+import { useQueryClient } from '@tanstack/react-query'
+import { useToast } from '@/_hooks'
+import { isErrorWithMessage } from '@/_utils'
+import { useDeleteTaskMutation } from './api'
+
+export const useDeleteTask = () => {
+  const queryClient = useQueryClient()
+  const { toast } = useToast()
+  const router = useRouter()
+
+  const { mutate, isPending } = useDeleteTaskMutation()
+
+  const handleSuccess = () => {
+    toast({ variant: 'success', title: 'Successfully deleted a task!' })
+    queryClient.invalidateQueries({
+      queryKey: ['todos', 'me', 'house', 'tasks'],
+    })
```

Comment:
It doesn't work here so please check it!

Reply:
Fixed it!
https://github.com/HAK2024/fairy-share/pull/86/commits/4000088f553e83a6dfcd0b7bceb18d2c241d5454